actions/kpops-runner/README.md

```diff
@@ -0,0 +1,14 @@
+# kpops-runner
+
+This action runs Kpops with the given config.
+
+## Input Parameters
+
+| Name                    | Required | Default Value |  Type  | Description                                                                                                   |
+| ----------------------- | :------: | :-----------: | :----: | ------------------------------------------------------------------------------------------------------------- |
+| mode |    ✅    |       -       | string | command mode used by Kpops (deploy, destroy, reset, clean)                                                |
```

Comment:
before merging we should format the table :)

Reply:
the table is now formatted :)

Reply:
it already looks much better, but still a little bit funky due to mixed indentation (mix of tabs & spaces). Could you run a formatter on it? Either https://github.com/dprint/dprint-plugin-markdown or https://github.com/prettier/prettier. If you don't have it it's fine and we can do it later

Reply:
PyCharm can also do that for you

Reply:
should be better now :)

Reply:
yep, better. did you use a formatter?

there's still a little bit mixed indentation in the `config` line but it's not a blocker.

Reply:
I used a formatter (one online and the pycharm one) and when I try to move it a little bit, it rollbacks to this state again.

Reply:
sure!